libraries/botbuilder-dialogs-adaptive/src/actions/replaceDialog.ts

```diff
@@ -16,14 +16,14 @@ export class ReplaceDialog<O extends object = {}> extends BaseInvokeDialog<O> {
     public constructor();
     
     /**
-     * Creates a new `ReplaceDialog` instance.
+     * Creates a new [ReplaceDialog](xref:botbuilder-dialogs-adaptive.ReplaceDialog) instance.
      * @param dialogId ID of the dialog to goto.
```

Comment:
Should we use the Dialog class reference here?, more lines with this point on this file.